ustwo-clockwise/src/main/java/com/ustwo/clockwise/wearable/permissions/WearablePermissionEducationActivity.java

```diff
@@ -0,0 +1,116 @@
+package com.ustwo.clockwise.wearable.permissions;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Color;
+import android.graphics.PorterDuff;
+import android.os.Bundle;
+import android.support.wearable.activity.ConfirmationActivity;
+import android.support.wearable.view.WatchViewStub;
+import android.view.View;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+import com.google.android.gms.wearable.DataMap;
+import com.ustwo.clockwise.R;
+import com.ustwo.clockwise.common.Constants;
+import com.ustwo.clockwise.common.WearableAPIHelper;
+import com.ustwo.clockwise.common.permissions.PermissionRequestItem;
+
+public class WearablePermissionEducationActivity extends Activity {
+
+    public static final String EXTRA_PERMISSION_REQUEST = "extra_permission_request";
+
+    private PermissionsRequest mPermissionsRequest;
+    private PermissionRequestItem mWearablePermission = null;
+
+    private WearableAPIHelper mWearableAPIHelper;
+    private boolean mAccepted = false;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        mWearableAPIHelper = new WearableAPIHelper(this, null);
+
+        setContentView(R.layout.permission_info);
+
+        mPermissionsRequest = PermissionsRequest.deserialize(getIntent().getByteArrayExtra(EXTRA_PERMISSION_REQUEST));
+        if(mPermissionsRequest == null || mPermissionsRequest.getRequestItems().size() == 0) {
+            throw new IllegalArgumentException();
```

Comment:
Can we maybe include a message with the exception, like that the intent needs to have an EXTRA_PERMISSION_REQUEST extra with (at least?) 1 item in it?